solidity/contracts/liquidity-protection/interfaces/ILiquidityProtection.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: SEE LICENSE IN LICENSE
+pragma solidity 0.6.12;
+import "./ILiquidityProtectionStore.sol";
+import "./ILiquidityProtectionStats.sol";
+import "./ILiquidityProtectionSettings.sol";
+import "../../token/interfaces/IERC20Token.sol";
+import "../../converter/interfaces/IConverterAnchor.sol";
+
+/*
+    Liquidity Protection interface
+*/
+interface ILiquidityProtection {
+    function store() external view returns (ILiquidityProtectionStore);
+    function stats() external view returns (ILiquidityProtectionStats);
+    function settings() external view returns (ILiquidityProtectionSettings);
+
+    function addLiquidityFor(
+        address _owner,
```

Comment:
These should be according to OZ convention, right?

Reply:
AFAIK, this PR (and task in general) is Stats migration, not changing coding conventions globally.

Reply:
It's a new interface... it's not refactoring of an old code